lib/livebook/hubs/enterprise.ex

```diff
@@ -8,25 +8,31 @@ defmodule Livebook.Hubs.Enterprise do
 
   @type t :: %__MODULE__{
           id: String.t() | nil,
```

Comment:
Btw, we can delete the enterprise hub stuff. It doesn't need to be in this PR, but perhaps the next one?

Reply:
Yeah, I was thinking on the next one!